packages/manager/modules/pci/src/projects/project/storages/databases/database.service.js

```diff
@@ -814,4 +814,30 @@ export default class DatabaseService {
       )
       .then(({ data }) => data);
   }
+
+  createConnectionPool(projectId, databaseId, payload) {
+    return this.$http
+      .post(
+        `/cloud/project/${projectId}/database/postgresql/${databaseId}/connectionPool`,
```

Comment:
**question:** Is there any reason to not pass an `engine` as parameter? Already done this way across this service.